pkg/sql/sqlbase/type_desc.go

```diff
@@ -255,24 +255,6 @@ func (desc *MutableTypeDescriptor) IsNew() bool {
 
 // AddEnumValue adds an enum member to the type.
 func (desc *MutableTypeDescriptor) AddEnumValue(node *tree.AlterTypeAddValue) error {
```

Comment:
Update the comment here now to say that "AddEnumValue assumes that the type is an enum, and that the new value doesn't exist already in the enum"

Reply:
updated comment, i will make sure to do such descriptive code comments in future contributions.